packages/lb-components/src/views/MainView/sidebar/index.tsx

```diff
@@ -199,14 +210,79 @@ const Sidebar: React.FC<IProps> = ({ sider, enableColorPicker }) => {
   }
 
   if (toolName === EPointCloudName.PointCloud) {
-    return (
-      <div className={`${sidebarCls}`}>
-        <div className={`${sidebarCls}__content`}>
-          <PointCloudToolSidebar />
-        </div>
-        <PointCloudOperation />
+    const showFoldSide = () => {
+      setShowSide(!showSide);
+      if (setSiderWidth) {
+        const w = showSide ? 48 : undefined;
+        setSiderWidth(w);
+      }
+    };
+
+    const hideSideBox = (
+      <div
+        style={{
+          width: '48px',
+          height: '100%',
+          display: 'flex',
+          justifyContent: 'center',
+          background: '#ffffff',
+        }}
+      >
+        <span onClick={() => showFoldSide()}>
+          <img
+            style={{
+              width: '16px',
+              height: '16px',
+              marginTop: '18px',
+              cursor: 'pointer',
+              transform: 'rotate(180deg)',
+            }}
+            src={menuFoldSvg}
+          />
+        </span>
       </div>
     );
+
+    return showSide ? (
```

Comment:
Don't use `Conditional (ternary) operator( ? ... : ... )`。 Use `if` is a better way to split logic